requirements.txt

```diff
@@ -1,4 +1,4 @@
-BeautifulSoup==3.2.1
+bs4==0.0.1
```

Comment:
Can you change this to `beautifulsoup4==4.5.1`? I didn't understand what `bs4` was at first (apparently it's a dummy package for Beautiful Soup).